press-permit-core.php

```diff
@@ -111,15 +112,15 @@ function () use ($min_wp_version) {
     if ($pro_active) {
         add_filter(
             'plugin_row_meta',
-            function($links, $file)
-            {
+            function ($links, $file) {
                 if ($file == plugin_basename(__FILE__)) {
-                    $links[]= __('<strong>This plugin can be deleted.</strong>', 'revisionary');
+                    $links[] = __('<strong>This plugin can be deleted.</strong>', 'revisionary');
```

Comment:
@richaferry At some point, this text domain should be changed, but it's not crucial for this release.

Reply:
I think this line changed coming from php formatter in my vscode @agapetry . I didn't changed that line intendeedly. Is it need i revert that line?

Reply:
It should pass 'press-permit-core' instead of 'revisionary'

Reply:
Ah i see. I got your point. nice catch @agapetry .

Reply:
Already fix in this commit https://github.com/publishpress/PublishPress-Permissions/pull/1313/commits/b3c22186398ef7fd8bf642ab9b5ecd3bd37caad8 @agapetry 
Thank you kevin. :+1: